modules/widgets/recurring-payments/form.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * Display the Recurring Payments widget Form.
+ *
+ * @package Jetpack
+ */
+
+?>
+<p>
+	<label for="<?php echo esc_attr( $this->get_field_id( 'title' ) ); ?>">
+		<?php esc_html_e( 'Widget Title', 'jetpack' ); ?>
+	</label>
+	<input
+		type="text"
+		class="widefat jetpack-recurring-payments-widget-title"
+		id="<?php echo esc_attr( $this->get_field_id( 'title' ) ); ?>"
+		name="<?php echo esc_attr( $this->get_field_name( 'title' ) ); ?>"
+		value="<?php echo esc_attr( isset( $instance['title'] ) ? $instance['title'] : '' ); ?>" />
+</p>
+<p class="jetpack-recurring-payments-products-fieldset" <?php if ( empty( $product_posts ) ) { echo 'style="display:none;"'; } ?>>
+	<label for="<?php echo esc_attr( $this->get_field_id( 'product_post_id' ) ); ?>">
+		<?php esc_html_e( 'Select previously created Recurring Payments plan:', 'jetpack' ); ?>
```

Comment:
I wonder if we should say "existing" here?

```suggestion
		<?php esc_html_e( 'Select an existing Recurring Payments plan:', 'jetpack' ); ?>
```